opentelemetry-sdk/src/logs/record.rs

```diff
@@ -186,17 +214,25 @@ mod tests {
         let mut log_record = LogRecord::default();
         let attributes = vec![(Key::new("key"), AnyValue::String("value".into()))];
         log_record.add_attributes(attributes.clone());
-        assert_eq!(log_record.attributes, Some(attributes));
+
+        let mut expected_attributes = AttributesGrowableArray::new();
```

Comment:
is it intentional that these tests rely on implementation detail, by relying on GrowableArray?

Reply:
Oh this is not really required, have removed now.